sql_script/create_tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS user (
+    user_id VARCHAR(64) PRIMARY KEY,
+    user_type VARCHAR(32) NOT NULL,
+    name VARCHAR(64)
+) ENGINE=InnoDB CHARACTER SET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+CREATE TABLE IF NOT EXISTS accounting (
+    id INT UNSIGNED PRIMARY KEY,
+    user_id VARCHAR(64) UNIQUE KEY,
+    kind VARCHAR(64),
+    created_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
why use two different time types to record?

Reply:
I totally forgot the reason but I'm going to change both of them to `datetime`  due to the limit of `timestamp`.